### PR TITLE
sync recipe with conda-forge and use updated fork from bundle-tools-2

### PR DESCRIPTION
**Pull request by jaimergp**
The "detect `setup.cfg` changes" workflow is not working (issues should have been created), so doing it manually now to catch up.

### CODE REVIEW

**Comment by Czaki** [comment]
is there an option for conda forge to check dependency on their side if setup.cfg or pyproject.toml is present?

**Comment by jaimergp** [comment]
`pip check` but it gives false positives due to the whole `PyQt5` vs `pyqt` dilemma. I think I can script something to avoid  the common false +s though, thanks for the suggestion!

File: .github/workflows/make_bundle_conda.yml [hunk 1]
```diff
@@ -401,21 +406,22 @@ jobs:
...
-      - name: Collect licenses
-        id: licenses
-        shell: bash -el {0}
-        working-directory: napari-packaging
-        run: |
-          licenses_zip_path=$(python build_installers.py --licenses)
-          echo "LICENSES_ARTIFACT_PATH=$licenses_zip_path" >> $GITHUB_ENV
-          echo "LICENSES_ARTIFACT_NAME=$(basename ${licenses_zip_path})" >> $GITHUB_ENV
-          echo "::set-output name=licenses_artifact::$licenses_zip_path"
+      # - name: Collect licenses
```
Comment (goanpeca) [inline .github/workflows/make_bundle_conda.yml:409]: is this pending a fix 🙃 ?

File: .github/workflows/make_bundle_conda.yml [hunk 2]
```diff
@@ -401,21 +406,22 @@ jobs:
...
-        uses: actions/upload-artifact@v2
-        with:
-          path: ${{ env.LICENSES_ARTIFACT_PATH }}
-          name: ${{ env.LICENSES_ARTIFACT_NAME }}
+
+      # - name: Upload License Artifact
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     path: ${{ env.LICENSES_ARTIFACT_PATH }}
+      #     name: ${{ env.LICENSES_ARTIFACT_NAME }}
```
Comment (goanpeca) [inline .github/workflows/make_bundle_conda.yml:424]: same here?

**Review by goanpeca — APPROVED** [review]

File: .github/workflows/make_bundle_conda.yml (hunk 2 above)
Comment (jaimergp) [reply .github/workflows/make_bundle_conda.yml:424]: In fact, I was not done and this shouldn't have been merged. The installers being generated now are generally broken in subtle ways til I fix some stuff on the constructor stack... not urgent or important enough to revert, but I'll address this during the week.

In general, if I have _not_ requested a review, this is not ready for review or merge, regardless the status of the PR. I'll open as draft from now on, but please double check before pulling the trigger.

**Pull request merged**